RPS_ver2.py

```diff
@@ -12,25 +18,25 @@ def decide_winner(p1, p2):
     # print 'player 1 chooses', player1
     # print 'player 2 chooses', player2
     if p1 == p2:
-        return "nobody! It's a tie"
-    if p1 == 'rock' and p2 == 'paper':
-        return 'Player Two'
-    if p1 == 'rock' and p2 == 'scissors':
-        return 'Player One'
-    if p1 == 'paper' and p2 == 'rock':
-        return 'Player One'
-    if p1 == 'paper' and p2 == 'scissors':
-        return 'Player Two'
-    if p1 == 'scissors' and p2 == 'rock':
-        return 'Player Two'
-    if p1 == 'scissors' and p2 == 'paper':
-        return 'Player One'
+        return "Nobody! It's a tie"
+    if p1 == rock and p2 == paper:
```

Comment:
note that we're using the variable paper and rock here instead of the string paper and rock - which you reproduced way too many times.  All it takes is one typo and your various validation and control steps are broken.